model/src/main/java/io/github/droidkaigi/confsched2018/model/Sponsor.kt

```diff
@@ -0,0 +1,22 @@
+package io.github.droidkaigi.confsched2018.model
+
+data class Sponsor(
+        var link: String,
+
+        var imageUri: ImageUri
+) {
+    sealed class ImageUri(val uri: String) {
+        class Base64ImageUri(uri: String) : ImageUri(uri)
+        class NetworkImageUri(uri: String) : ImageUri(uri)
```

Comment:
I think we can create data class extends a sealed class. What do you think? 👍

Reply:
let me try!

Reply:
👍 https://github.com/DroidKaigi/conference-app-2018/pull/176/commits/d3e5fbda64bf45ba1a6f41ad9de7abbf589d8a29